src/scenes/AcceleratorRouter/AcceleratorDeliverablesList.tsx

```diff
@@ -80,9 +82,6 @@ const AcceleratorDeliverablesList = () => {
   const [filters, setFilters] = useState<Record<string, SearchNodePayload>>({});
   const [searchValue, setSearchValue] = useState('');
   const debouncedSearchTerm = useDebounce(searchValue, 250);
-  // Logging this out for now so the build doesn't fail, still needs to be hooked up per previous TODO
-  // tslint:disable-next-line:no-console
-  console.log(debouncedSearchTerm);
```

Comment:
The build was previously failing without this `console.log` statement..? Interesting!

Reply:
It was failing with it, that's why I had to add the tslint ignore line. I added it because I assumed it would fail without using `debouncedSearchTerm`... but your other comment makes me think it wouldn't have mattered? I will look into it tomorrow, ideally it fails for console logs and unused variables.

Reply:
Ah logging to avoid an unused variable flag makes sense.